tests/test_create.py

```diff
@@ -358,135 +358,125 @@ def test_safety_checks_disabled(
 
 
 def test_json_create_install_update_remove(
-    path_factory: PathFactoryFixture,
+    tmp_path: Path,
     conda_cli: CondaCLIFixture,
-    capsys: CaptureFixture,
 ):
     # regression test for #5384
 
-    def assert_json_parsable(content):
-        string = None
-        try:
-            for string in content and content.split("\0") or ():
+    def is_json_parsable(content: str) -> bool:
```

Comment:
Can we simply rename this test like that? 🤔 
Do we not need to deprecate it applying the deprecation policy?

Reply:
I think you mean it's a local func and inaccessible outside the test! :D 
Makes sense! 👍🏼